static/js/show-task.js

```diff
@@ -2,6 +2,8 @@ var tasks;
 var worklist_name;
 var worklist_created_by;
 var input;
+var activeDropdownElementIndex;
+var activeElementIndex;
```

Comment:
Not a huge deal here, but let's avoid global variables if we can? How about making `rememberState` return an Object with these two values, and then we pass that to `restoreState` later?

Reply:
Done.